vhost_user_net/src/lib.rs

```diff
@@ -94,7 +94,7 @@ impl std::convert::From<Error> for std::io::Error {
 
 pub struct VhostUserNetBackend {
     mem: Option<GuestMemoryMmap>,
-    vring_worker: Option<Arc<VringWorker>>,
+    vring_worker: Option<Vec<Arc<VringWorker>>>,
```

Comment:
Plural

Reply:
No change it since I do not see the benefit. Is there any potential issue if not remove Option?